adserver/reports.py

```diff
@@ -288,7 +293,11 @@ def get_index_display(self, index):
         """Handle making sure dates use the same 3 letter syntax as Django."""
         if isinstance(index, datetime.date):
             # %a is the day of week, so you know which are weekends
-            return index.strftime("%b %d, %Y (%a)")
+            value = index.strftime("%b %d, %Y")
+            # Make it so these parse as dates in a CSV
+            if not self.export:
+                value += index.strftime(" (%a)")
```

Comment:
If we're already going to differentiate, I'd just export as ISO8601 and then keep the above format for display. If we want to get really fancy, we could use an [HTML `time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

Reply:
Yea, something to think more about. Don't think we need to address it here.